performance_regression_test.py

```diff
@@ -255,6 +255,21 @@ def preload_data(self, compaction_strategy=None):
         else:
             self.log.warning("No prepare command defined in YAML!")
 
+        if post_prepare_cql_cmds := self.params.get('post_prepare_cql_cmds'):
+            self.log.debug("Execute post prepare queries: %s", post_prepare_cql_cmds)
+            self._run_cql_commands(post_prepare_cql_cmds)
+
```

Comment:
Maybe it's better to implement this in Teser class ? Or another shared place, and not repeat this code multiple times ?

Reply:
o.k.